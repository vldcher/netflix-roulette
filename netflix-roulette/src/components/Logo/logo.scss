@import '../../shared/palette.scss';

.logo {
    position: relative;
    color: $primary;
    font-weight: normal;
    z-index: 1;
    font-size: 16px;

    &--bold {
        font-weight: bold;
    }

    @media (min-width: 768px) {
        font-size: 20px;
    }
}