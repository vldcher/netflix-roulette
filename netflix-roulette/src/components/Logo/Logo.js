import React from "react";
import './logo.scss';

export default () => {

  return (
    <h3 className="logo"><span className="logo--bold">netflix</span>roulette</h3>
  );
};