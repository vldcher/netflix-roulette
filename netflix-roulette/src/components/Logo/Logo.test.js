import React from 'react';
import renderer from 'react-test-renderer';
import Logo from './Logo';

it('renders correctly Logo', () => {
  const logo = renderer.create(<Logo />).toJSON();
  expect(logo).toMatchSnapshot();
});