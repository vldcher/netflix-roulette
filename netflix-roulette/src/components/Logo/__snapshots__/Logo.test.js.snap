// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Logo 1`] = `
<h3
  className="logo"
>
  <span
    className="logo--bold"
  >
    netflix
  </span>
  roulette
</h3>
`;
